sigs/runtime-charter.md

```diff
@@ -0,0 +1,150 @@
+set out by the CNCF TOC](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#responsibilities--empowerment-of-sigs).
+
+
+## Current CNCF Projects considered to be within the Scope of this SIG
+
+
+
+1. Kubernetes
+2. Containerd
+3. Harbor
+4. Dragonfly
+5. Virtual Kubelet
+6. CRI-O
+7. KubeEdge
+8. KubeVirt
+
+
+## Interfaces With Other Related Groups
+
+
+
+*   **[Several Kubernetes SIGs](https://github.com/kubernetes/community)**
+    cover Kubernetes-specific workload, scheduling, autoscaling, execution
+    and other related abstractions, interfaces, and implementations of
+    these interfaces.  We will maintain communication with these Kubernetes
+    SIGs where needed.  Our aim is to avoid unnecessary duplication of
+    effort by the two groups and maintain clear and consistent messaging
+    to our end user community and projects.
+*   **[CNCF Security SIG](https://github.com/cncf/sig-security)**
+    - works on the more general area of cloud-native security including a
+    uthentication, authorization, encryption, accounting, auditing, and
+    related topics.  We defer as much as possible to this group to deal
+    with general security-related issues and liaise closely with them on
+    how to deal with security areas where these arise.
+*   **[CNCF App Delivery SIG](https://github.com/cncf/sig-app-delivery)**
+    is focussed on the development, deployment, operation and testing of
+    cloud-native applications.  We collaborate with this SIG where it
+    pertains to helping to ensure that the required underlying workload
+    execution abstractions and mechanisms are suitably provided to support
+    these application-level delivery needs.
+*   **Note regarding Container Registries/Repositories** and the like:
+    While image packaging and distribution (and hence container
+    registries/repositories in general) fall within the scope of this
+    Runtime SIG, many of their common features and use cases are better
+    dealt with by other CNCF SIGS.  For example:
+    *   image storage, caching, etc - Storage SIG
+    *   Image encryption, signing etc - Security SIG
+    *   use of image registries to store and distribute many other types
+        of artifacts, and in particular the format of these artifacts,
+	including helm charts, OPA policies, public security certificates,
+	data sets, machine learning models, etc, etc - the SIG relevant to
+	those artifact types, e.g. Apps SIG, Security SIG, etc.
+
+
+## Operating Model
+
+This SIG follows the [standard operating guidelines](https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#operating-model)
+provided by the TOC unless otherwise stated here.
+
+**Current TOC Liaison:**  Brendan Burns and Brian Grant
+
+**[Proposed] Co-Chairs:** [Quinton Hoole](https://www.linkedin.com/in/quintonhoole/),
+  [Ricardo Aravena](https://www.linkedin.com/in/raravena), 1 TBD
+
+**[Proposed] Tech Leads:** [Klaus Ma](http://www.klaus1982.cn/about/), 2 TBD
```

Comment:
Hi, I'm Pengfei Ni and I'm interested in being the co tech-lead of sig-runtime. I'm a software engineer at Microsoft Azure and maintainer of the kubernetes project. I have extensive experience in Cloud Computing, Kubernetes and Software Defined Networking (SDN). I have given presentations on KubeCon China/NA, ArchSummit, LC3, and so on. And here is my linkedin https://www.linkedin.com/in/feisky/.

Reply:
I am also interested in being a tech lead for sig-runtime. I am a Principal Engineer at Red Hat. I am an early libcontainer maintainer, OCI maintainer, and runc maintainer. I lead the CRI-O project. I have contributed to docker and kubernetes and I'm an active participant in sig-node. I have given talks at Dockercon, Kubecon and Linux Plumbers Conference. Happy to provide more details if needed. Thanks!

Reply:
Thanks all.

@feiskyer I've added you as a nominee (if nobody else has an objection)

@mrunalp can you sync up with @umohnani8 (also about @quinton-hoole's question above) since only one of you can be nominated due to both of you being at Red Hat, (2/3 supermajority discouraged) and only one more tech-lead nominee slot available.

Elections rules described here:  https://github.com/cncf/toc/blob/master/sigs/cncf-sigs.md#elections

Reply:
@quinton-hoole @raravena80 sorry for the late response. Thanks for providing more clarification around what the needs would be for a tech lead in this sig.  Please nominate @mrunalp, but I am looking forward to actively being involved with the sig.